arch/riscv/CMakeLists.txt

```diff
@@ -2,4 +2,10 @@
 
 add_subdirectory(core)
 
+if(CONFIG_64BIT)
+zephyr_compile_options(-mabi=lp64 -march=rv64imac -mcmodel=medany)
```

Comment:
remove zephyr_compile_options() from this file, its sufficient to be in target.cmake